core/arch/arm/plat-imx/main.c

```diff
@@ -134,8 +134,12 @@ static void main_fiq(void)
 
 void console_init(void)
 {
+#ifdef CONSOLE_UART_BASE
 	imx_uart_init(&console_data, CONSOLE_UART_BASE);
 	register_serial_console(&console_data.chip);
+#else
+	register_serial_console(NULL);
+#endif
```

Comment:
The `#else` part seems unnecessary (the `static struct serial_chip *serial_console` in `core/kernel/console.c` is initialized to NULL).

Reply:
You're right, I will remove the #else part.